networking/networking.go

```diff
@@ -229,18 +233,41 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
-	if len(n.nets) == 0 {
-		return nil
+// GetDefaultNet iterates through all loaded networks and returns either
+// the first network that has masquerading enabled,
+// or the last network in case there is no masqueraded one,
+// or an error if no network was loaded.
```

Comment:
Also, the name of the function `GetDefaultNet` is very generic does not say much about what it does and how it does it.

GetNetForPortForwarding / GetIPForPortForwarding?

@s-urbaniak might have better ideas without getters.

Alternatively, since this special network ordering is only for port forwarding, shouldn't they be moved to networking/portfwd.go?